wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder.swift

```diff
@@ -81,7 +87,18 @@ extension EventDecoder {
             lastIndex = StoredUpdateEvent.highestIndex(self.eventMOC)
 
             guard let index = lastIndex else { return }
-            decryptedEvents = self.decryptAndStoreEvents(filteredEvents, startingAtIndex: index)
+
+            if DeveloperFlag.proteusViaCoreCrypto.isOn {
+                decryptedEvents = self.decryptAndStoreEvents(
+                    filteredEvents,
+                    startingAtIndex: index
+                )
+            } else {
+                decryptedEvents = self.legacyDecryptAndStoreEvents(
+                    filteredEvents,
+                    startingAtIndex: index
+                )
+            }
```

Comment:
What's the strategy regarding tests here? I didn't see any changes in `EventDecoderTests` that would cover the decryption through core crypto

Reply:
Yeah it's true I forgot to add theses tests, I'll add them.

Reply:
Done